src/Chainweb/Chainweb.hs

```diff
@@ -487,15 +490,15 @@ withChainwebInternal conf logger peer serviceSock rocksDb pactDbDir resetDb inne
         { _cutDbParamsLogLevel = Info
         , _cutDbParamsTelemetryLevel = Info
         , _cutDbParamsUseOrigin = _cutIncludeOrigin cutConf
-        , _cutDbParamsInitialHeightLimit = _cutInitialCutHeightLimit cutConf
+        , _cutDbParamsInitialHeightLimit = _cutInitialBlockHeightLimit cutConf
         }
       where
         cutConf = _configCuts conf
 
     synchronizePactDb :: HM.HashMap ChainId (ChainResources logger) -> CutDb cas -> IO ()
     synchronizePactDb cs cutDb = do
         currentCut <- _cut cutDb
-        mapConcurrently_ syncOne $ mergeCutResources $ _cutMap currentCut
+        traverse_ syncOne $ mergeCutResources $ _cutMap currentCut
```

Comment:
Does this synchronize pact dbs for chains sequentially? That would be very slow. What is the reason for this change?